.github/workflows/check_website.yaml

```
@@ -63,3 +63,15 @@ jobs:
       # info: https://etok.codes/acmcsuf.com/blob/main/CONTRIBUTING.md#npm-bloat
       - name: Test Dependency Size
         run: npm run test:size
+
+      - name: Transform Genuary Check Changes
+        id: transform-genuary-check-changes
+        run: if git diff --exit-code --quiet -- scripts/transform-genuary; then
+          echo "updated=0" >> $GITHUB_OUTPUT
+          else
+          echo "updated=1" >> $GITHUB_OUTPUT
+          fi
```

Apologies, I missed this syntax error in my previous review. This text is multiline.

```suggestion
        run: |
           if git diff --exit-code --quiet scripts/transform-genuary; then
             echo "updated=0" >> $GITHUB_OUTPUT;
           else
             echo "updated=1" >> $GITHUB_OUTPUT;
           fi
```